hash/ibm5170.xml

```diff
@@ -21838,6 +22156,34 @@ license:CC0
 		</part>
 	</software>
 
+	<!-- PC Review (UK) -->
+
+	<software name="pcr199205">
+		<description>PC Review (Issue 07 - May 1992)</description>
+		<year>1992</year>
+		<publisher>EMAP Images</publisher>
+		<notes>Disk includes: "Another World (demo)", "Time Lord (demo)", "Aces of the Pacific (preview pictures)"</notes>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="PC Review (Issue 07 - May 1992) [EMAP] [1992] [3.5DD] [Disk 1 of 1].img" size="737280" crc="f6db2032" sha1="82a5cb528c7e6ba08f0daf0e981119ec2ff16bc8"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pcr199311">
+		<description>PC Review (Issue 25 - November 1993)</description>
+		<year>1993</year>
+		<publisher>EMAP Images</publisher>
+		<notes>Disk includes: "Ancients: Death Watch (demo)"</notes>
+		<!-- Missing Disk A -->
```

Comment:
Should you add a `NO_DUMP` part for the missing disk?  It’s almost invisible like this.

Reply:
Excellent idea! Corrected.